spec/models/form/subsection_spec.rb

```diff
@@ -69,22 +51,14 @@
     end
 
     context "with optional fields" do
-      subject(:subsection) { described_class.new(subsection_id, subsection_definition, section) }
-
-      let(:section_id) { "tenancy_and_property" }
-      let(:section_definition) { form.form_definition["sections"][section_id] }
-      let(:section) { Form::Section.new(section_id, section_definition, form) }
-      let(:subsection_id) { "property_information" }
-      let(:subsection_definition) { section_definition["subsections"][subsection_id] }
-
       it "has a started status even if only an optional field has been answered" do
-        lettings_log.postcode_known = 0
+        lettings_log.age1 = nil
```

Comment:
I'm confused by this test. Is age1 optional? If not what is the optional field that has been answered?

Reply:
Yeah, good point, I'll update it